packages/shared/getComponentName.js

```diff
@@ -19,6 +21,10 @@ import {
   REACT_STRICT_MODE_TYPE,
   REACT_PLACEHOLDER_TYPE,
 } from 'shared/ReactSymbols';
+import {
+  refineResolvedThenable,
+  getResultFromResolvedThenable,
+} from 'react-reconciler/src/ReactFiberLazyComponent';
```

Comment:
Not sure what the rules are for deep linking from the `shared` directory... I can inline the helpers if needed.

Reply:
Doesn't look good. If those are meant to be sharable then they should be in `shared` themselves. Like `TypeOfWork`.